gk/main.c

```diff
@@ -558,32 +559,17 @@ gk_del_flow_entry_from_hash(struct rte_hash *h, struct flow_entry *fe)
 	return ret;
 }
 
-/* Return true when it removes at least one entry, and false otherwise. */
+/* Return true when it removes an entry, and false otherwise. */
 static bool
-gk_flow_tbl_bucket_scan(uint32_t *bidx, struct gk_instance *instance)
+gk_flow_tbl_entry_scan(uint32_t entry_idx, struct gk_instance *instance)
```

Comment:
`gk_flow_tbl_entry_scan()` would be better suited receiving `fe` instead of `entry_idx` since it doesn't take advantage of `entry_idx`. Another side benefit of this change is that the caller of `gk_flow_tbl_entry_scan()` can take advantage of the computation of the address of `fe` (i.e. `&instance->ip_flow_entry_table[entry_idx]`) for the prefetch and save it in a local variable.